distribution/docs/src/main/resources/_contents/running-contents.adoc

```diff
@@ -322,6 +322,9 @@ Provides a list of environment variables.
 |`spatial`
 |Searches spatially the local Catalog.
 
+|`validate`
+|Runs a specified file against all applicable validators.
```

Comment:
I think you're right. I'll also change the command description to use that word too.